button.go

```diff
@@ -96,7 +96,7 @@ func (r *buttonRenderer) Refresh() {
 	r.text.Refresh()
 	if r.b.key.Icon != r.icon.File {
 		r.icon.File = r.b.key.Icon
-		r.icon.Refresh()
+		go r.icon.Refresh()
```

Comment:
This should not need a go launch, it is just queuing a refresh event in our canvas.